src/multi-input/index.jsx

```diff
@@ -85,7 +90,8 @@ export default function MultiInput({ value, onChange, onFieldChange, name, label
               name={!objectItems && name}
               onRemove={removeItem(item.id)}
               onChange={updateItem(item.id)}
-              disabled={disabled}
+              isDisabled={disabled.includes(item.id)}
```

Comment:
Do the disabled check here rather than passing all the disabled ids to every Item (not a functional change, just a stylistic one).